src/Settings.js

```diff
@@ -82,57 +83,124 @@ const ClearStorage = inject("store")(
 );
 
 const Settings = inject("store", "morsePlayer")(
-  observer(({ store, morsePlayer }) => (
-    <div>
-      <Helmet>
-        <title>Settings</title>
-      </Helmet>
-      <h1>Settings</h1>
-      <div>
-        <h2>Morse tone</h2>
-        <div>
-          <Slider
-            id="speed"
-            label="Speed (WPM)"
-            editable
-            max={80}
-            min={10}
-            value={store.morse.speed}
-            onChange={value => store.morse.setSpeed(value)}
-            leftIcon={<FontIcon>fast_forward</FontIcon>}
-          />
-          <Slider
-            id="frequency"
-            label="Frequency (Hz)"
-            editable
-            max={1000}
-            min={200}
-            step={10}
-            value={store.morse.frequency}
-            onChange={value => store.morse.setFrequency(value)}
-            leftIcon={<FontIcon>audiotrack</FontIcon>}
-          />
-          <Button
-            raised
-            primary
-            className="md-block-centered"
-            iconEl={<FontIcon>play_arrow</FontIcon>}
-            onClick={() => {
-              event("test");
-              morsePlayer.playString("hello");
-            }}
-          >
-            Test
-          </Button>
-        </div>
+  observer(
+    class Settings extends Component {
+        playCount = 0;
+        playInterval;
 
-        <h2>Internals</h2>
-        <div>
-          <ClearStorage />
-        </div>
-      </div>
-    </div>
-  ))
+        playLoop = () => {
+            if(!this.props.store.morse.playing) {
```

Comment:
Please run https://prettier.io/ over your branch so that code style is consistent.

(I'm working on setting it up as a git commit hook and if possible within Travis as well)